torch/_inductor/fx_passes/overlap_preserving_bucketer.py

```diff
@@ -396,6 +404,7 @@ def _find_buckets(
             # Greedy optimization: stop after consecutive failures
             consecutive_failures = 0
             max_consecutive_failures = 20
+            start_node_idx = self.node_idx[start_node]
```

Comment:
can we iterate in local vicinity first ? e.g 1, then -1, then 2, then -2. 

The reason we dont do that previously is we start at beginning, so we dont need to try to bucket nodes that have already been attempted

Reply:
From offline discussion: decided to keep bucketing in more exposed first order with node distance limitation to get more signal about performance on different models.